src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/WindowsForms/WindowsFormsEditorProvider.cs

```diff
@@ -117,6 +117,7 @@ public async Task<bool> SetUseGlobalEditorAsync(string documentMoniker, bool use
             IProjectTreeServiceState result = await _projectTree.Value.TreeService.PublishAnyNonLoadingTreeAsync();
 
             if (result.TreeProvider.FindByPath(result.Tree, documentMoniker) is IProjectItemTree treeItem &&
+                treeItem.Parent != null &&
```

Comment:
Behaviour change. Parent is nullable, though in practice these items should always be parented. Avoids NRE.